Error while installing version 3.0.18

## Steps to reproduce
<!-- Describe the operations you have performed -->
Install version `3.0.18` of `isbot`.

## Expected behaviour
<!-- Describe what you expected to happen -->
Module should be installed successfully.

## Actual behaviour
<!-- Describe what actually occurred -->
Getting the following error:
```
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! isbot@3.0.18 postinstall: `./scripts/download-fixtures.sh`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the isbot@3.0.18 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

--- a/.gitignore
+++ b/.gitignore
@@ -4,6 +4,7 @@ tests/fixtures/user-agents.net.bot-crawler.txt
 tests/fixtures/live_webcrawlers.txt
 tests/fixtures/crawler-user-agents-monperrus.json
 tests/fixtures/matomo-bots.yml
+tests/fixtures/downloaded
 tests/.browser-tests.js
 docs
 contributors.json

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,6 @@
+# 3.0.19
+- Fix: It's not needed to download fixtures at postinstall
+
 # 3.0.18
 - Add [AngleSharp parser](https://github.com/AngleSharp/AngleSharp)
 - Some Maintenance

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "isbot",
-  "version": "3.0.18",
+  "version": "3.0.19",
   "description": "🤖 detect bots/crawlers/spiders via the user agent.",
   "keywords": [
     "bot",
@@ -24,7 +24,6 @@
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
-    "postinstall": "./scripts/download-fixtures.sh",
     "prebuild": "which parcel || npm i parcel-bundler --no-save",
     "build": "parcel build page/index.pug --out-dir docs --public-url .",
     "postbuild": "echo isbot.js.org > docs/CNAME",
@@ -33,6 +32,7 @@
     "prestart": "which parcel || npm i parcel-bundler --no-save",
     "start": "parcel page/index.pug --out-dir docs",
     "sort": "./scripts/sort.js",
+    "preunit": "[ -f \"tests/fixtures/downloaded\" ] || ./scripts/download-fixtures.sh",
     "unit": "node --experimental-worker ./node_modules/.bin/mocha tests/*.js --exclude 'tests/.*'",
     "prebrowser": "rollup --config .rollup.js",
     "browser": "karma start .karma.js",

--- a/scripts/download-fixtures.sh
+++ b/scripts/download-fixtures.sh
@@ -9,3 +9,5 @@ curl -f https://raw.githubusercontent.com/matomo-org/device-detector/master/Test
 
 echo "Download crawler list from crawler-user-agents repo"
 curl -f https://raw.githubusercontent.com/monperrus/crawler-user-agents/master/crawler-user-agents.json > tests/fixtures/crawler-user-agents-monperrus.json
+
+date -u > tests/fixtures/downloaded

